docs/source/configuration/configuration-partition.rst

```diff
@@ -0,0 +1,51 @@
+.. _configurations-partition:
+
+Graph Partition
+============================
+
+For users who are already familiar with DGL and know how to construct DGL graph, GraphStorm provides two graph partition tools to partition DGL graphs into the required input format for GraphStorm launch tool for training and inference.
+
+* `partition_graph.py <https://github.com/awslabs/graphstorm/blob/main/tools/partition_graph.py>`_: for Node/Edge Classification/Regress task graph partition.
+* `partition_graph_lp.py <https://github.com/awslabs/graphstorm/blob/main/tools/partition_graph_lp.py>`_: for Link Prediction task graph partition.
+
+`partition_graph.py <https://github.com/awslabs/graphstorm/blob/main/tools/partition_graph.py>`_ arguments
+---------------------------------------------------------------------------------------------------------------
+
+- **--dataset**: (**Required**) the graph dataset name defined for the saved DGL graph file.
+- **--filepath**: (**Required**) the file path of the saved DGL graph file.
+- **--target_ntype**: the node type for making prediction, required for node classification/regression tasks. This argument is associated with the node type having labels. Current GraphStorm supports **one** predict node type only.
```

Comment:
```suggestion
- **--target-ntype**: the node type for making prediction, required for node classification/regression tasks. This argument is associated with the node type having labels. Current GraphStorm supports **one** predict node type only.
```

Reply:
All arguments of the graph_partition.py and graph_partition_lp.py are still using "_", not "-". 

So, I am not sure if we need to change those arguments before v0.1 release.